### PR TITLE
Add new variables in backend/generalhw.pm

### PR DESCRIPTION
**Pull request by jeeeunevans-ct**
This commit adds new variables in generalhw backend which makes
more configurable.

### CODE REVIEW

File: backend/generalhw.pm [hunk 1]
```diff
@@ -87,7 +87,8 @@ sub relogin_vnc ($self) {
         'vnc-base',
         {
             hostname => $bmwqemu::vars{GENERAL_HW_VNC_IP} || die('Need variable GENERAL_HW_VNC_IP'),
-            port => 5900,
+            port => defined($bmwqemu::vars{GENERAL_HW_VNC_PORT}) ? $bmwqemu::vars{GENERAL_HW_VNC_PORT} : 5900,
+            password => defined($bmwqemu::vars{GENERAL_GW_VNC_PASSWORD}) ? $bmwqemu::vars{GENERAL_GW_VNC_PASSWORD} : undef,
```
Comment (okurz) [inline backend/generalhw.pm:91]:
```suggestion
            port => $bmwqemu::vars{GENERAL_HW_VNC_PORT} // 5900,
            password => $bmwqemu::vars{GENERAL_GW_VNC_PASSWORD},
```

**Review by okurz — CHANGES_REQUESTED** [review]
Hi jeeeunevans-ct, welcome to this project. Great idea! Just some minor suggestions

File: backend/generalhw.pm [hunk 2]
```diff
@@ -87,7 +87,8 @@ sub relogin_vnc ($self) {
         'vnc-base',
         {
             hostname => $bmwqemu::vars{GENERAL_HW_VNC_IP} || die('Need variable GENERAL_HW_VNC_IP'),
-            port => 5900,
+            port => $bmwqemu::vars{GENERAL_HW_VNC_PORT} // 5900,
+            password => $bmwqemu::vars{GENERAL_GW_VNC_PASSWORD},
```
Comment (perlpunk) [inline backend/generalhw.pm:91]: Typo? `GENERAL_GW_VNC_PASSWORD` vs `GENERAL_HW_VNC_PASSWORD`

**Review by okurz — COMMENTED** [review]
Looks good. Now please squash the commits into one

**Review by Martchus — APPROVED** [review]

**Review by okurz — APPROVED** [review]

**Pull request merged**